pipeline/baremetal/5.3.yaml

```diff
@@ -0,0 +1,18 @@
+---
+-
+  cluster_conf: conf/quincy/upi/octo_5_node_env.yaml
+  name: "Tier-1: End to End scenarios executed on greenfield deployment"
+  os_version: 8.6
+  platform: rhel-8
+  rhbuild: "5.3"
+  scenario: 1
+  stages:
+    deploy:
+      abort_on_failure: true
+      suites:
+        - suites/quincy/e2e/poc/test_cluster_deployment.yaml
+    baseline:
+      suites:
+        - suites/quincy/e2e/poc/test-cephfs-core-features.yaml
+        - suites/quincy/e2e/poc/test-rbd-core-features.yaml
+        - suites/quincy/e2e/poc/test-object-core-features.yaml
```

Comment:
Let's soft link to `pacific`... `test-object-core-features.yaml` would require some changes. That is branch would change to `ceph-pacific` in the test config